ranger/gui/color.py

```diff
@@ -30,6 +31,12 @@ def get_color(fg, bg):
         size = len(COLOR_PAIRS)
         try:
             curses.init_pair(size, fg, bg)
+        except ValueError:
+            # We're trying to add more pairs than the terminal can store,
+            # approximating to the closest color pair that's already stored
+            # would be cool but the easier solution is to just fall back to the
+            # default fore and background colors, pair 0
+            COLOR_PAIRS[key] = 0
```

Comment:
Shouldn't the function return `COLOR_PAIRS[key]` right after this line?
Right now it just skips to line 54 after catching the exception and the value of `COLOR_PAIRS[key]` ends up being assigned to whatever `size` is at the moment, instead of `0`.

Reply:
I've just pushed a fix, thanks a lot for noticing! @toonn, please check if you're fine with my fix.

Reply:
Yes, thanks to the both of you!